app/src/main/java/org/dhis2/usescases/events/ScheduledEventPresenterImpl.kt

```diff
@@ -112,4 +104,17 @@ class ScheduledEventPresenterImpl(
             .uid()
         d2.eventModule().events().uid(eventUid).setAttributeOptionComboUid(catOptComboUid)
     }
+
+    fun stageHasExtraInfo(): Boolean {
+        val event = d2.eventModule().events().uid(eventUid)
+            .blockingGet()
+        val stage = d2.programModule().programStages().uid(event.programStage())
+            .blockingGet()
+        val program = d2.programModule().programs().uid(stage.program()?.uid())
+            .blockingGet()
+        val hasCoordinates = stage.featureType() != null && stage.featureType() != FeatureType.NONE
+        val hasNonDefaultCatCombo = d2.categoryModule().categoryCombos()
+            .uid(program.categoryComboUid()).blockingGet().isDefault != true
+        return hasCoordinates || hasNonDefaultCatCombo
+    }
```

Comment:
does it make sense to move it to SDK ???

Reply:
Could be yes, i'll move it to to the logic class

Reply:
great ! can you move it to the dhis2 logic package? huge thanks !